android/cli/commands/_build.js

```diff
@@ -2269,7 +2269,14 @@ AndroidBuilder.prototype.generateAppProject = async function generateAppProject(
 			logger: this.logger.debug
 		});
 	}
-
+	
+	const huaweiServicesFile = path.join(this.projectDir, 'platform', 'android', 'agconnect-services.json');
+ 	if (await fs.exists(huaweiServicesFile)) {
+ 		afs.copyFileSync(huaweiServicesFile, path.join(this.buildAppDir, 'agconnect-services.json'), {
```

Comment:
<!--
  1 failure:  android/cli/comma...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: android/cli/commands/_build.js;
  Line: 2275;
-->
- :no_entry_sign: android/cli/commands/_build.js line 2275 – Mixed spaces and tabs. (no-mixed-spaces-and-tabs)